devnet.sh

```diff
@@ -20,13 +20,16 @@ build_binary=${build_binary:-y}
 read -p "Do you want to clear the existing ledger history? (y/n, default: n): " clear_ledger
 clear_ledger=${clear_ledger:-n}
 
+# Log verbosity is set to 1 (DEBUG) by default.
+verbosity=1
+
 if [[ $build_binary == "y" ]]; then
   # Ask the user if they want to enable validator telemetry
   read -p "Do you want to enable validator telemetry? (y/n, default: y): " enable_telemetry
   enable_telemetry=${enable_telemetry:-y}
 
   # Build command
-  build_cmd="cargo install --locked --path ."
+  build_cmd="cargo install --locked --path . --offline --debug"
```

Comment:
Can the offline option be an issue if you don't have any of the crates downloaded yet?

Reply:
@kaimast I think we should remove both of these new options:
- running with `--debug` will make everything much less performant (especially as VM crypto is involved), not an ideal default. 
- running with `--offline` can cause issues if dependencies are not cached by the user.

Reply:
Good catch. I added them to speed up compilation and forgot to remove them.